tests/admin_scripts/tests.py

```diff
@@ -2152,6 +2152,52 @@ def test_custom_project_template_with_non_ascii_templates(self):
                 'Some non-ASCII text for testing ticket #18091:',
                 'üäö €'])
 
+    def test_custom_project_template_hidden_directory_default_excluded(self):
+        "Make sure hidden directories are excluded by default"
```

Comment:
We try to avoid boilerplate like "make sure" or "ensure that." Two instances below, also.

Also it's true that the single double-quote is used elsewhere in this file, but I don't think it's current style.

```suggestion
        """Hidden directories are excluded by default."""
```

Reply:
Thanks, fixed.